src/components/SimpleSelect.js

```diff
@@ -37,19 +38,21 @@ export default class SimpleSelect extends Component<Props<*>> {
     })
 
     return (
-      <Select
-        classNamePrefix="react-select"
-        options={options}
-        value={options.find(o => o.value === value)}
-        onChange={e => {
-          onChange(e ? e.value : null)
-        }}
-        styles={NORMAL_SELECT_STYLES}
-        isDisabled={disabled}
-        placeholder={placeholder}
-        isClearable={isClearable}
-        menuPortalTarget={document.body}
-      />
+      <div data-dw={this.props['data-dw']}>
```

Comment:
since you're wrapping this `Select` component in a `div` here to assign the `data-dw` (soon to be `data-test` prop), why not just not pass the `data-test` prop and wrap the `SimpleSelect` component in a `div` where it's invoked?

Reply:
I'm not sure if I understand exactly what you mean. Are you saying like why not wrap where the SimpleSelect component gets instantiated with a div Instead of passing in a `data-test` prop and wrapping the `select` with a div?

Reply:
yep, that's exactly what I'm saying. just to avoid passing a `data-test` prop if we don't have to. would that not work?

Reply:
oh ok I see, that should also work.

Doing it that approach would mean that I would have to wrap each instantiation of `SimpleSelect` with a div IF I wanted to test it whereas in the current implementation I just need to wrap `Select` with a div once and then pass an optional prop whenever I want to test another `SimpleSelect`. 

A pro of doing it that approach is like you said we would avoid passing an extra prop.

I'm not sure what's the better alternative in this case, what do you think?

Reply:
For sure. The it's written now, the `div` will always be there, even if `data-dw` is undefined. Which would result in unnecessary `div`s in the DOM for places that we aren't using that particular element for testing.

I am of the opinion that the `div` should be around `SimpleSelect`, and not within `SimpleSelect`, but I'll leave the decision to you and encourage you to do what you think is best.

Reply:
oh interesting I never considered that. I think the approach you suggested is better then 👍